backend/service/aws/dynamodb.go

```diff
@@ -2,22 +2,46 @@ package aws
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 
 	dynamodbv1 "github.com/lyft/clutch/backend/api/aws/dynamodb/v1"
+	awsv1 "github.com/lyft/clutch/backend/api/config/service/aws/v1"
 )
 
+// defaults for the dynamodb settings config
+const (
+	AwsMaxRCU       = int64(40000)
+	AwsMaxWCU       = int64(40000)
+	SafeScaleFactor = float32(2.0)
```

Comment:
In a follow up PR lets make sure to comment the defaults in the configuration proto for the reader.
https://github.com/lyft/clutch/blob/0a6e4a00e72b8bb3a976f801d02f9550adf447d7/api/config/service/aws/v1/aws.proto#L26-L27

Reply:
Will follow up with updated proto file that includes comments and echo fields